roles/openshift_master/tasks/main.yml

```diff
@@ -131,6 +131,23 @@
 - name: Install the systemd units
   include: systemd_units.yml
 
+- name: Checking for journald.conf
+  stat: path=/etc/systemd/journald.conf
+  register: journald_conf_file
+
+- name: Increase rate limits in journald
+  lineinfile:
+    dest: /etc/systemd/journald.conf
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}"
+  with_items:
+    - { regexp: '^#?RateLimitInterval=.*$', line: 'RateLimitInterval=1s' }
+    - { regexp: '^#?RateLimitBurst=.*$', line: 'RateLimitBurst=10000' }
```

Comment:
Also, this hardcodes the rate we want to be 10,000 log entries a second.  It would be best if the user could deploy and update this setting with a value they can configure to accommodate their needs.  In the future 10,000/s might be too small, while in other scenarios 10,000/s might be too large.